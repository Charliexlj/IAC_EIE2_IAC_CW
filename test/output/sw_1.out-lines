register_v0 = aaaaaaaa
