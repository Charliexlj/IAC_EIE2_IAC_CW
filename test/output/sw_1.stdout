RAM : INIT : Loading RAM contents from test/hex/sw_1.hex.txt
TB : finished; active=0
register_v0 = aaaaaaaa
rtl/mips_cpu_bus_tb.v:62: $finish called at 43000 (10ps)
